packages/runtime/src/parser/parser.ts

```diff
@@ -267,7 +286,7 @@ export async function parser (program: string, interpreter: Interpreter): Promis
   let counter = 0;
 
   const lines = program.split('\n');
-  for await (const line of lines) {
```

Comment:
i think we can remove Promise as well. can you please confirm ?

Reply:
yes